chef/cookbooks/tempest/templates/default/tempest_cleanup.sh.erb

```diff
@@ -4,6 +4,16 @@ export OS_AUTH_URL=<%= @keystone_settings['internal_auth_url'] %>
 export OS_AUTH_STRATEGY=keystone
 export OS_REGION_NAME='<%= @keystone_settings['endpoint_region'] %>'
 
+<% if keystone_settings['insecure'] -%>
+function keystone()
+{
+    command keystone --insecure "$@"
+}
+<% end -%>
```

Comment:
So to be honest, I find it weird that we sometimes try to only use the insecure mode when needed, and sometimes we just use it.

As I said, I would prefer if we only use it when needed, but in the other case, we should at least be consistent and just always use it.